corehq/apps/users/models.py

```diff
@@ -2820,6 +2836,7 @@ class UserReportingMetadataStaging(models.Model):
     commcare_version = models.TextField(null=True)
     build_profile_id = models.TextField(null=True)
     last_heartbeat = models.DateTimeField(null=True)
+    fcm_token = models.TextField(null=True)
```

Comment:
Not sure why other fields are Text fields. 
I decided to go ahead with TextField in case current `fcm_token` length changes in future as there is no offiical information on it,